data/scripts.js

```diff
@@ -1232,7 +1232,7 @@ exports.BattleScripts = {
 					if (hasMove['dracometeor']) rejected = true;
 					break;
 				case 'return':
-					if (hasMove['bodyslam'] || hasMove['facade'] || hasMove['doubleedge'] || hasMove['tailslap']) rejected = true;
+					if (hasMove['bodyslam'] || hasMove['facade'] || hasMove['doubleedge'] || hasMove['tailslap'] || hasMove['doublehit']) rejected = true;
```

Comment:
I think you meant to change Doubles. It will never be needed here so you can remove it from here.